Composer/packages/client/src/components/CreationFlow/v2/TemplateDetailView.tsx

```diff
@@ -44,25 +53,73 @@ type TemplateDetailViewProps = {
   readMe: string;
 };
 
+const templateDocUrl = 'https://aka.ms/localComposerTemplateDoc';
+
 export const TemplateDetailView: React.FC<TemplateDetailViewProps> = (props) => {
+  const { setLocalTemplatePathState } = useRecoilValue(dispatcherState);
+
+  const localTemplatePath = useRecoilValue(localTemplatePathState);
+  const isLocalTemplate = props.template?.id === localTemplateId;
   // Composer formats and displays its own template title and strips out title from read me to avoid redundant titles
   const getStrippedReadMe = () => {
     return props.readMe.replace(/^(#|##) (.*)/, '').trim();
   };
 
+  const validatePath = async (path): Promise<string> => {
+    const response = await httpClient.get(`/storages/validate/${encodeURI(path)}`);
+    return response.data.errorMsg;
+  };
+
+  const renderLocalTemplateForm = () => (
+    <Fragment>
+      <Text>
+        {formatMessage.rich(
+          `To create a bot from your own <templateDocLink>Bot Framework Template</templateDocLink> you need to add a path to your local templates index.js file. <templateDocLink>Learn more</templateDocLink>`,
```

Comment:
nit: what's the point of having two links with the same source right next to each other, is the first of necessary?

Reply:
Removed